### ISSUE TITLE
Syntax error in generated pydot code on 20.04 + Noetic

### ISSUE THREAD

**Issue by PeterMitrano** [ros-visualization/rqt_tf_tree#23]
ros-noetic-rqt-tf-tree version  0.6.1-1focal.20200826.185237
My python interpreter is 3.8
pydot version is 1.4.1
ros-noetic-qt-dotgraph is at 0.4.2-1focal.20200812.174341
python3-yaml is version  5.3.1-1

Runnning rqt_tf_tree gives me this error.

```
load for full details.
  data = yaml_parser.load(yaml_data)
"dot" with args ['-Tdot', '/tmp/tmpl5iq8obx'] returned code: 1

stdout, stderr:
 b''
b"Error: /tmp/tmpl5iq8obx: syntax error in line 196 near ':'\n"

PluginHandlerDirect._restore_settings() plugin "rqt_tf_tree/RosTfTree#1" raised an exception:
Traceback (most recent call last):
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_gui/plugin_handler_direct.py", line 121, in _restore_settings
    self._plugin.restore_settings(plugin_settings_plugin, instance_settings_plugin)
  File "/opt/ros/noetic/lib/python3/dist-packages/rqt_tf_tree/tf_tree.py", line 131, in restore_settings
    self._refresh_tf_graph()
  File "/opt/ros/noetic/lib/python3/dist-packages/rqt_tf_tree/tf_tree.py", line 143, in _refresh_tf_graph
    self._update_graph_view(self._generate_dotcode())
  File "/opt/ros/noetic/lib/python3/dist-packages/rqt_tf_tree/tf_tree.py", line 150, in _generate_dotcode
    return self.dotcode_generator.generate_dotcode(dotcode_factory=self.dotcode_factory,
  File "/opt/ros/noetic/lib/python3/dist-packages/rqt_tf_tree/dotcode_tf.py", line 98, in generate_dotcode
    self.dotcode = self.dotcode_factory.create_dot(self.graph)
  File "/opt/ros/noetic/lib/python3/dist-packages/qt_dotgraph/pydotfactory.py", line 175, in create_dot
    dot = graph.create_dot()
  File "/usr/lib/python3/dist-packages/pydot.py", line 1726, in new_method
    return self.create(
  File "/usr/lib/python3/dist-packages/pydot.py", line 1953, in create
    assert process.returncode == 0, process.returncode
AssertionError: 1
```

**Comment by PeterMitrano** [comment]
Here's the offending tmp file, maybe that would help?

[tmpuao_ks81.txt](https://github.com/ros-visualization/rqt_tf_tree/files/5245959/tmpuao_ks81.txt)

**Comment by AchmadFathoni** [comment]
Any update on this?

**Comment by PeterMitrano** [comment]
Seems fixed as of `0.6.2-1focal.20210922.190701`
